docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+---
+version: '3'
+services:
+  consumer:
+    image: golang:alpine
```

Comment:
executing the binary the imagine could be from scratch tho

Reply:
added a multistage build, last stage will use from scratch